controllers/provisioners/eks/scaling/launchconfig.go

```diff
@@ -282,6 +293,17 @@ func (lc *LaunchConfiguration) blockDeviceList(volumes []v1alpha1.NodeVolume) []
 	return sortConfigDevices(devices)
 }
 
+func (lc *LaunchConfiguration) metadataOptions(input *v1alpha1.MetadataOptions) *autoscaling.InstanceMetadataOptions {
```

Comment:
Current pattern is to place these kind of constructors in awsprovider, could we do the same in this case? There is no real need for this to be a method of LaunchConfiguration -  would be good if we can merge #296 first to avoid the conflict